code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -39,15 +39,16 @@
 		update_stat("adjustBrainLoss")
 	return STATUS_UPDATE_STAT
 
-/mob/living/carbon/human/setBrainLoss(amount, updating = TRUE)
+/mob/living/carbon/human/setBrainLoss(amount, updating = TRUE, ignore_brain_mod = FALSE)

```

Comment:
I know it's not really on you since it was like that before but man, the duplicate code between setbrainloss and adjustbrainloss really bugs me. As does the fact now one has an additional var/check while the other one doesn't.

Reply:
Yeah that's true. I suppose I can add this arg to adjustBrainLoss as well. I'm sure it will also be useful at some point